src/uct/ib/base/ib_md.h

```diff
@@ -153,6 +153,10 @@ typedef struct uct_ib_md {
      * be initiated.  */
     uint32_t                 flush_rkey;
     uint16_t                 vhca_id;
+    struct {
+        uint32_t base;
+        size_t   size;
```

Comment:
is there a reson for them to be different types?

Reply:
fixed, using uint32_t